Build breakage observed downstream

We observed the following in the SAW CI run:
```
COQC theories/Ref/EnTreeSpecFacts.v
File "./theories/Ref/EnTreeSpecFacts.v", line 432, characters 60-101:
Error: No such goal. The current bullet * is unfinished.

make[1]: *** [theories/Ref/EnTreeSpecFacts.vo] Error 1
```

Something must have changed upstream, as nothing changed here.

--- a/theories/Ref/EnTreeSpecCombinatorFacts.v
+++ b/theories/Ref/EnTreeSpecCombinatorFacts.v
@@ -233,7 +233,7 @@ Proof.
    + pstep. constructor. auto.
  - repeat rewrite bind_tau. pstep. constructor.  pclearbot. left.
    eapply refines_iter_aux; eauto.
- - repeat rewrite bind_vis. pstep. constructor. auto. pclearbot. left.
+ - repeat rewrite bind_vis. pstep. constructor. auto. pclearbot. intros * H5. left.
    apply H2 in H5. pclearbot.
    eapply refines_iter_aux; eauto.
  - rewrite bind_tau. pstep. constructor. pstep_reverse.

--- a/theories/Ref/EnTreeSpecFacts.v
+++ b/theories/Ref/EnTreeSpecFacts.v
@@ -429,7 +429,10 @@ Proof.
     + remember (VisF (Spec_vis e2) k2) as x.
       hinduction REL before r; intros; inv Heqx; inj_existT; subst.
       * constructor; auto. intros. eapply H0 in H2. right. pclearbot.
-        eapply CIH; eauto with solve_padded. 2 : apply REL. 2 : destruct H2; try contradiction; auto. 
+        eapply CIH; eauto with solve_padded.
+        2 : apply REL.
+        (* these bits no longer needed 20241014 *)
+        (* 2 : destruct H2; try contradiction; auto. *)
         inv Hpad1. inj_existT. subst. pstep.
         constructor. auto.
       * constructor. eapply IHREL; eauto. inv H1. pclearbot. pstep_reverse.

